modules/logging-vector-fluentd-feature-comparison.adoc

```diff
@@ -95,8 +95,3 @@ include::snippets/logging-outputs-5.6-snip.adoc[]
 | Log event buffering   | &#10003; |
```

Comment:
Lokistack on IPv6  are fixed recently https://issues.redhat.com/browse/LOG-4570. Let me confirm if that had been backport to 5.5.

Reply:
I'm not sure this is really within the scope of the PR?